setup.py

```diff
@@ -15,7 +15,7 @@
     "tox",
 ]
 
-if sys.version_info < (3, 5):
+if sys.version_info < (3, 4):
```

Comment:
I'm confused by this change. Do the tests not work with Python 3.5 and those Django versions?